README.md

```diff
@@ -58,10 +58,11 @@ You'll want to run the site locally to test your changes.
 
 **Clone the source code, build, and serve:**
 
-3. Clone this repository and `cd` into it.
-4. Run `bundle install`
-5. Run `bundle exec jekyll serve`
-6. Browse the site at http://127.0.0.1:4000/
+3. Clone this repository `git clone https://github.com/BitcoinDesign/Guide.git` 
```

Comment:
I think this is only applicable to people who have direct commit access. Most editors will be working from their own fork of the Guide and therefore a different URI for the repo.

Reply:
I mean, sure, if I followed this instructions, it would absolutely work. But presumably if they are running a local copy of the Guide, they want to make edits and commits, which would require a fork.

Reply:
Good point!